README.md

```diff
@@ -34,16 +34,26 @@ Jordan can also remind you of these things.
 7. discuss how crazy the media is in times of corona. 
 8. discuss how hard it is to be in self-isoltation.
 
+**Flow Chart**
+
+![Jordan Output Flow](https://user-images.githubusercontent.com/64072862/81957953-92366100-960d-11ea-9d37-abc82e5fd2e9.png)
 
 **My Project**
 
-First, I will implement some welcome string that introduces Jordan. 
-In this first opening conversation, there will be some if-else-statements. 
-My goal for this project is to make a chatbot that can give people comfort in times of corona.  
-What I am cutting, if I am short on time is to not implement all modules (conversation types) that I am planning now. 
-But, what I will do, when I have extra time, is to add more modules and things to discuss, and just make Jordan a bit more advanced. 
+My goal for this project is to make a chatbot that can give people comfort in times of corona. What I am cutting, if I am short on time is to not implement all modules (conversation types) that I am planning now. But, what I will do, when I have extra time, is to add more modules and things to discuss, and just make Jordan a bit more advanced. 
+
+Implementation Steps
+1. Create a flow chart of Jordan's output. Actually I did this already, and you can see it above.
+2. Create a function that decodes a sentence grammatically such that it will use an appropriate response. For this step, there might be several smaller functions to be created, for instance, one that converts the grammar aspects like changing from first-person to second-person perspective.
+3. Create an answers library that includes keywords Jordan is supposed to react to. For example, when someone writes 'I need chocolate', 'need', would be such a keyword that Jordan can respond to with 'Why do you think you need chocolate?'?
+4. Create a function that initiates an introduction conversation. This introduction conversation is going to consist of a series of questions that come from a library in which introduction sentences are stored, such as "What is your name?" or "How are you today?"
+5. Create a function, in which the participant can pick a topic to talk about. 
+6. Create an open-question library within the example topic classes. There will be one for cursing with Jordan, one for small talk with Jordan, etc. 
```

Comment:
I love the idea! I think you already did a great job mapping out the steps and building blocks you have to work on in order to fulfill the project. It's very impressive! I was just thinking that this seems more like a project for a PhD Thesis to me (considering its size) than for a 5 weeks course.
One comment about what to cut if you're short on time: maybe there are already packages out there that can grammatically decode input sentences. I think that this is the most challenging part because in order for the right response to work (at least this is how I imagine it), you not only have to teach the software all the words possible and  which response they are supposed to trigger but also based on which word or words it' supposed to react, how they change meaning depending on the remaining sentence, maybe differences between verbs and nouns (to need vs. a need) and maybe some other aspects I can't even think about.
So maybe I would check whether there is code already somewhere that does all of this and then for your project, focus on what answers Jordan should provide and implement that as well/or on the graphical user interface.